src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- [ ]  Your App should have unit tests for several components of your app and also if you you should implement test for pure functions from your redux codes if they are there 

- [ ]  You should have an integration test using React Testin Library as well 
YOu might perform mocking data fetching, etc ...

![image](https://user-images.githubusercontent.com/43877689/183241609-84285a7f-f201-45b9-8912-b364fe8e9367.png)